packages/next-tinacms-cloudinary/src/cloudinary-tina-cloud-media-store.ts

```diff
@@ -0,0 +1,36 @@
+/**
+Copyright 2021 Forestry.io Holdings, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CloudinaryMediaStore } from './cloudinary-media-store'
+import { Client } from 'tina-graphql-gateway'
+export class TinaCLoudCloudinaryMediaStore extends CloudinaryMediaStore {
+  client: any
+  constructor(client: Client) {
+    super()
+    this.client = client
+    console.log('child class')
+    this.fetchFunction = async (input: RequestInfo, init?: RequestInit) => {
+      try {
+        const url = input.toString()
+        const query = `${url.includes('?') ? '&' : '?'}org=${
+          client.organizationId
+        }&clientID=${client.clientId}`
+
+        const res = client.fetchWithToken(url + query, init)
```

Comment:
Why is this calling Tina Cloud's client? I would expect this to hit Cloudinary directly with no need for our client at all

Reply:
I think we can remove the cloudinary API entirely since we are handling everything on the backend for now.  We need to use the Tina Cloud Client to get access to the access token, client ID, and org ID. This is what authenticates the request.